src/utils.ts

```diff
@@ -492,9 +492,9 @@ export function now(): number {
 }
 
 /** @internal */
-export function calculateDurationInMs(started: number): number {
+export function calculateDurationInMs(started: number | undefined): number {
```

Comment:
I also didn't find where started could be set as `undefined`.

Reply:
Isn't it fine for a let to remain unassigned? typescript will know if you use it before assignment right?